LICENSE

```diff
@@ -64,7 +64,7 @@ DAMAGE.
 
 --------------------------------------------------------------------------------
 
-Submodule taken from the multipledispatch project (https://github.com/mrocklin/multipledispatch)
+Submodule that were taken from the multiple dispatch project (https://github.com/mrocklin/multipledispatch)
```

Comment:
`Submodules taken from the multiple-dispatch project...`

Reply:
thanks for suggestion @smichr